metadata.json

```diff
@@ -77,6 +60,14 @@
     {
       "name": "puppetlabs-concat",
       "version_requirement": ">= 2.0.0 < 6.0.0"
+    },
+    {
+      "name": "puppetlabs-apt",
```

Comment:
does this qualify as a soft dependency?

Reply:
Well, in Puppet 6 that all changes as neither yumrepo isn't included in core anymore. So any module wanting to manage a yum or apt repo will now have to make those modules dependencies (or write their own).

Reply:
> They are repackaged back into Puppet agent, so you do not have to install them separately

https://puppet.com/docs/puppet/6.0/type.html#supported-types-in-modules

Reply:
https://puppet.com/docs/puppet/6.0/style_guide.html#dependencies

> Soft dependencies should be called out in the README.md, and must not be enforced as a hard requirement in your metadata.json